docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md

```diff
@@ -0,0 +1,71 @@
+
+If you need some guidance, please watch the D/HD workshop
+[video](https://deakin365.sharepoint.com/:v:/r/sites/ThothTech2/Shared%20Documents/General/Recordings/2-4-22%20D-HD%20Workshop.mp4?csf=1&web=1&e=0Tomse).
+It is beneficial for everyone.
+
+Please look ahead of Task 6.x. There are 4 tasks that due on the same day on Sun
+24/04/2022.
+
+## Company tasks
+
+- Please fill in the learning
+  [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=7Hgj0IgW1UaFQBwotfRw9pxmFQ5mP6pJsCNkD-HdouBUNUVBVlBEMlVTT0RKWkdVNTJOT0o5MTNMTSQlQCN0PWcu)
+  to help us understand the company learning and development needs and plan for
+  resources.
+- Please review the Quality Assurance (QA)
+  [document](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/quality-assurance-overview.md)
+  and start a conversation around the topic with your team. There will be a
+  session on this during the week (Wed 6/04/2022 6:00 PM - 6:45 PM) and it is
+  open to everyone.
+- Please review the Communication
+  [Guide](https://github.com/thoth-tech/handbook/blob/main/docs/communication/communication.md)
+  and see if it can help to streamline your team communication process.
+- Any seniors who want to mentor, please reach out to the Leadership team.
+- Any juniors who would like to work towards leadership next trimester. We are
+  organising an Expression of Interest for attending the leadership conference.
+  Please stay tuned for more information.
+- Please work on the relevant project repositories from **the Thoth Tech
+  organisation**. Please do not pull or push to Doubtfire or SplashKit
+  repositories.
+
+## What you can expect
+
+- Showcase for D/HD students is being organised at the end of the week. We would
+  like to present project Trello boards, plans, testing & data strategies to the
+  Managing Directors. It would be a great opportunity to receive feedback.
+- We are making transition away from MS Word docs to Thoth-tech
+  [handbook](https://github.com/thoth-tech/handbook) repository on GitHub
+  starting from this week onward.
+- We are planning for more training in the next weeks so please watch out for
+  invites in your mailbox or subscribe to the
+  [shared calendar](https://outlook.office.com/calendar/group/deakin365.onmicrosoft.com/_thoth-tech/view/month).
+
+## What we can expect
+
+- We expect that Trello boards should be planned out and upskilling should
+  already be happening. If you need help, please reach out
+- Team members should feel comfortable with GitHub and creating pull requests.
+  Please review the training
+  [video](https://deakin365.sharepoint.com/:v:/r/sites/ThothTech2/Shared%20Documents/General/Recordings/24-3-22%20Github%20Repos%20QnA.mp4?csf=1&web=1&e=Pa3Hh0)
+  and the QA
+  [document](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/quality-assurance-overview.md)
+  if you are still unsure.
```

Comment:
If you have done both of these things and still feel lost - please reach out to the area leads